app/lib/features/chat/widgets/in_chat.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:acter/features/chat/pages/room_page.dart';
+import 'package:acter/features/chat/pages/room_profile_page.dart';
+import 'package:acter/features/chat/providers/chat_providers.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+class InChat extends ConsumerStatefulWidget {
+  final Widget child;
+  const InChat({Key? key, required this.child}) : super(key: key);
+
+  @override
+  ConsumerState<InChat> createState() => _InChatConsumerState();
+}
+
+class _InChatConsumerState extends ConsumerState<InChat>
+    with AutomaticKeepAliveClientMixin {
+  @override
+  Widget build(BuildContext context) {
+    super.build(context);
+    final showFullView = ref.watch(showFullSplitView);
+    final convo = ref.watch(currentConvoProvider);
+    return LayoutBuilder(
+      builder: (context, constraints) {
+        return Row(
```

Comment:
did you ever test this on a  completely fresh empty account without any chats yet? what does that look like?

Reply:
No. But It think it should reflect the empty placeholder for overview. But for ```RoomPage()```, we might want to show empty or just hide it.